src/Transfer/Resources/Database/Attribute.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Database;
+
+use Utopia\Transfer\Resource;
+use Utopia\Transfer\Transfer;
+
+abstract class Attribute extends Resource
+{
+    public const TYPE_STRING = 'stringAttribute';
```

Comment:
```suggestion
    public const TYPE_STRING = 'string';
```

Let's make sure we use consistent naming conventions. For example in this case, we don't need to use `Attribute` as a suffix for each constant. 

Additionally In the Resource.php file we do 

https://github.com/utopia-php/transfer/pull/4/files#diff-88659cac19f8979c53b5ae2a7f6ed026b9a71198b7c1556b718cf2ff6d586d81R27-R35

Where the constant values are title case. Let's stay consistent and use lower case for the constant value everywhere